Add September newsletter (#125)
readthedocs/blog: newsletter-september-2021.rst

.. post:: September 2, 2021
   :tags: newsletter, python
   :author: Juan Luis
   :location: MAD

.. meta::
   :description lang=en:
      Company updates and new features from last month,
      current focus, and upcoming features in September.

Read the Docs newsletter - September 2021
=========================================

Welcome to the latest edition of our monthly newsletter, where we
share the most relevant updates around Read the Docs,
offer a summary of new features we shipped
during the previous month,
and share what we'll be focusing on in the near future.

Company highlights
------------------

> - We have finally published

Reviewer: ```suggestion
- We have published
```

Not sure we want to use `finally` about ourselves :D

  :doc:`the first release candidate of version 1.0.0 of our Sphinx theme </theme-release-100rc1>`,
  which adds support for recent versions of Sphinx and docutils among other things,
  and announced our future plans for it.
  Check out the linked blog post to know more.

> - The :doc:`first part of the new Read the Docs tutorial <readthedocs:tutorial/index>`,

Reviewer: These both feel more like features to me (eg. `major features`), instead of company updates (eg. staffing or similar). Not sure what to do about it tho, but it makes our new features section feel really weak.

Author: Yes, the concept of "highlights" is a bit broad. Since the first one has a related blog post and is quite important and the second is grant work, and in the absence of a better suggestion, I'll leave them like this.

  which we wrote as part of :doc:`our CZI grant </czi-grant-announcement>`, is online!

>   We hope that it serves as a better starting point for people seeking to learn how to use our platform.

Reviewer: ❤️

New features
------------

- We added the possibility for users to `remove themselves from a
  project <https://github.com/readthedocs/readthedocs.org/pull/8384>`_
  without having to ask an owner to remove them
  (as long as they are not the only owner of the project).
- We fixed `a small usability issue with our search-as-you-type
  box <https://github.com/readthedocs/readthedocs-sphinx-search/pull/93>`_,
  that ignored the first typed characters under certain circumstances.
- Commercial users can now `share specific versions of their
  projects <https://docs.readthedocs.io/en/stable/commercial/sharing.html>`_
  and `log out from the flyout

>   menu <https://docs.readthedocs.io/en/stable/versions.html#logging-out>`_.

Reviewer: I'd put this higher, it seems like the most useful feature.

Author: Done!

- We have documented our `unofficial support for
  Gitea <https://github.com/readthedocs/readthedocs.org/pull/8402>`_,
  and made other minor documentation fixes.

Thanks to our external contributors `Mozi`_, `Maksudul Haque`_,
`Stefano Costa`_, and `Christian Clauss`_.

You can always see the latest changes to our platforms in our `Read the Docs
Changelog <https://docs.readthedocs.io/page/changelog.html>`_.

.. _Mozi: https://github.com/pzhlkj6612
.. _Maksudul Haque: https://github.com/saadmk11
.. _Stefano Costa: https://github.com/steko
.. _Christian Clauss: https://github.com/cclauss

Upcoming features
-----------------

- Ana_ will continue researching tools for conducting automated testing on our Sphinx theme,
  review pull requests corresponding to the 1.1 milestone,
  and make some small styling improvements to our documentation.
- Anthony_ will work with Ana_ on testing our Sphinx theme,
  in addition to resuming work on our new user interface
  and doing some financial updates.

> - Eric_ will keep pushing updates on our Commercial landing page
>   and continue working on our sales process.

Reviewer: ```suggestion
- Eric_ will keep pushing updates on our Commercial landing page
  and continue working on our sales processes. He's also 
  working to continue building EthicalAds as well. 
```

  without having to ask an owner to remove them
  (as long as they are not the only owner of the project).
- We fixed `a small usability issue with our search-as-you-type
  box <https://github.com/readthedocs/readthedocs-sphinx-search/pull/93>`_,
  that ignored the first typed characters under certain circumstances.
- Commercial users can now `share specific versions of their
  projects <https://docs.readthedocs.io/en/stable/commercial/sharing.html>`_
  and `log out from the flyout
  menu <https://docs.readthedocs.io/en/stable/versions.html#logging-out>`_.
- We have documented our `unofficial support for
  Gitea <https://github.com/readthedocs/readthedocs.org/pull/8402>`_,
  and made other minor documentation fixes.

Thanks to our external contributors `Mozi`_, `Maksudul Haque`_,
`Stefano Costa`_, and `Christian Clauss`_.

You can always see the latest changes to our platforms in our `Read the Docs
Changelog <https://docs.readthedocs.io/page/changelog.html>`_.

.. _Mozi: https://github.com/pzhlkj6612
.. _Maksudul Haque: https://github.com/saadmk11
.. _Stefano Costa: https://github.com/steko
.. _Christian Clauss: https://github.com/cclauss

Upcoming features
-----------------

> .. TBC after the next sprint planning

Reviewer 2: ```suggestion
```

  :doc:`the first release candidate of version 1.0.0 of our Sphinx theme </theme-release-100rc1>`,
  which adds support for recent versions of Sphinx and docutils among other things,
  and announced our future plans for it.
  Check out the linked blog post to know more.
- The :doc:`first part of the new Read the Docs tutorial <readthedocs:tutorial/index>`,
  which we wrote as part of :doc:`our CZI grant </czi-grant-announcement>`, is online!
  We hope that it serves as a better starting point for people seeking to learn how to use our platform.

New features
------------

- We added the possibility for users to `remove themselves from a
  project <https://github.com/readthedocs/readthedocs.org/pull/8384>`_
  without having to ask an owner to remove them
  (as long as they are not the only owner of the project).
- We fixed `a small usability issue with our search-as-you-type
  box <https://github.com/readthedocs/readthedocs-sphinx-search/pull/93>`_,
  that ignored the first typed characters under certain circumstances.
- We made several documentation updates and improvements
  about `commercial project sharing <https://github.com/readthedocs/readthedocs.org/pull/8239/>`_,

>   `logging out <https://github.com/readthedocs/readthedocs.org/pull/8286/>`_,

Author: Cool, thanks! Done

  and `unofficial support for Gitea <https://github.com/readthedocs/readthedocs.org/pull/8402>`_,
  among other minor fixes.

Thanks to our external contributors `Mozi`_, `Maksudul Haque`_,
`Stefano Costa`_, and `Christian Clauss`_.

You can always see the latest changes to our platforms in our `Read the Docs
Changelog <https://docs.readthedocs.io/page/changelog.html>`_.

.. _Mozi: https://github.com/pzhlkj6612
.. _Maksudul Haque: https://github.com/saadmk11
.. _Stefano Costa: https://github.com/steko
.. _Christian Clauss: https://github.com/cclauss

Upcoming features
-----------------

.. TBC after the next sprint planning

- Ana_
- Anthony_
- Eric_
- `Juan Luis`_
- Manuel_
- Santos_

Possible issues
---------------

We have been `making changes to how we store cookies <https://github.com/readthedocs/readthedocs.org/pull/8304>`_
to make our site more secure.
This has caused some minor issues in certain web browsers

> or to `users that were embedding our pages inside an

Author: Thanks!
